lib/mailbox.lua

```diff
@@ -40,12 +105,24 @@ function MailBox:receive(timeout)
 	if header_size == 0 then
 		return nil
 	elseif header_size < sizeoft then
-		error("malformed message")
+		error("malformed message: header too small")
```

Comment:
let's not mix code change in this PR, please.. it will make it harder for tracking changes